fava/application.py

```diff
@@ -124,6 +125,21 @@ def _inject_filters(endpoint, values):
             values[filter_name] = g.filters[filter_name]
 
 
+@app.url_defaults
```

Comment:
Since we only require this in two places (or really only one right now), I think it would be better to just to this explicitly in the two places instead of having this code that runs everytime that `url_for` is called anywhere. It could be done two ways: 1) add a function for it 2) add a template filter for mtime and then do it in the template

Reply:
I think I'd prefer 1)